bin/mdsctl

```diff
@@ -264,7 +264,7 @@ where app in:
   mds-audit
   mds-compliance
   mds-daily
-  mds-geography
+  mds-geography-author
```

Comment:
Why are we renaming this?

Reply:
I talked with Neil and it seems that we should have a `geography-author` and `geography` service to mirror `policy` and `policy-author`.